profiler/src/main/java/com/splunk/opentelemetry/profiler/RecordingFileNamingConvention.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+class RecordingFileNamingConvention {
+
+  private static final String PREFIX = "otel-profiler";
+  private final Path outputDir;
+
+  public RecordingFileNamingConvention(Path outputDir) {
```

Comment:
Nit: the class is package-private, public modifiers on methods don't mean anything

Reply:
I hear ya.  I guess in general I'm not a huge fan of restricting access, and I think that pulling `public` off of methods muddies the object shape / user contract.  It's much harder to quickly look and see "what does this object provide" if everything is just package access, and it's made worse when some methods are package access just to facilitate testing.

I understand that this is the convention in this repo (and upstream) so I'll try and be better about keeping consistent.